docs/releases/v1.15.4-2.md

```diff
@@ -0,0 +1,262 @@
+# Release notes
+
+- [Release notes](#release-notes)
+  - [Auto-join nodes feature](#auto-join-nodes-feature)
+    - [Use case](#use-case)
+    - [Issue](#issue)
+    - [Change](#change)
+      - [How to deploy it](#how-to-deploy-it)
+        - [Before start](#before-start)
+        - [Requirements](#requirements)
+        - [Upgrading an existing cluster](#upgrading-an-existing-cluster)
+          - [Terraform part](#terraform-part)
+          - [Ansible part](#ansible-part)
+        - [Verify](#verify)
+
+This version contains a new feature: **auto-join** nodes.
+
+## Auto-join nodes feature
+
+### Use case
+
+Having a kubernetes cluster with nodes belonging to an autoscaling group. Node pool has to be able to create and destroy in a dynamic way nodes. New nodes needs to join the kubernetes cluster without human intervention.
```

Comment:
I would write "Node pool has to be able to create and destroy nodes in a dynamic way." Moreover, I would capitalize the word "kubernetes".

Reply:
Solved